examples/custom-resources/foreign-namespace-upstreams/README.md

```diff
@@ -0,0 +1,107 @@
+# Upstreams in foreign namespaces
+
+In this example we use the [VirtualServer and
+VirtualServerRoute](https://docs.nginx.com/nginx-ingress-controller/configuration/virtualserver-and-virtualserverroute-resources/)
+resources to configure load balancing for the modified cafe application from the [Basic
+Configuration](../basic-configuration/) example. We have put the load balancing configuration as well as the deployments
+and services into multiple namespaces. Instead of one namespace, we now use three: `tea`, `coffee`, and `cafe`.
```

Comment:
@haywoodsh does it makes sense to include a suggestion around including these ns in -watch-namespace flag if not already?